server/controllers/kanban/card.js

```diff
@@ -0,0 +1,16 @@
+const { Card } = require("../../models/Card");
+const asyncHandler = require("express-async-handler");
+
+exports.getCard = asyncHandler(async (req, res, next) => {
+  if (!req.params.cardId) {
+    res.status(400);
+    throw new Error("Card ID is required");
```

Comment:
instead of throwing an error, you can include it in your response for example
`res.status(400).send({ error: "Card ID is required" });`

Reply:
i think throwing errors are great for logging as part of error handling, although it may also be beneficial for your frontend to know why it's returning a 400

same thing for the other responses

Reply:
Hello, I thought so as well however, the express-async-handler seems to handle the json response for this. With the thrown Error, I still received a json response on the response.